controllers/cloud.redhat.com/providers/utils/utils.go

```diff
@@ -154,3 +164,55 @@ func MakeLocalDBService(s *core.Service, nn types.NamespacedName, baseResource o
 func MakeLocalDBPVC(pvc *core.PersistentVolumeClaim, nn types.NamespacedName, baseResource obj.ClowdObject, capacity string) {
 	utils.MakePVC(pvc, nn, providers.Labels{"service": "db", "app": baseResource.GetClowdName()}, capacity, baseResource)
 }
+
+// GetCaddyImage returns the caddy image to use in a given environment
+func GetCaddyImage(env *crd.ClowdEnvironment) string {
```

Comment:
Can we put these in the web provider?

Reply:
Sure. Still not clear what you mean when you say put these in the web provider though.

You want them to be receiver methods on that provider? Or just locate these funcs in the web provider package and adjust the imports everywhere else?